NewForce-at-Mountwest/bangazon-api-bucatini
# Roo

# Description

All models are generated and characterized.
Database is seeded.
Controllers are created; customer controller get method has been base coded


## Related Ticket(s)
Ticket #1


## Steps to Test
Open solution, check for all necessary models and needed characteristics.
Need assistance for build error in class Startup, line 45 "app.UseMVC()"


# Checklist:
- [ N ] I have commented my code, particularly in hard-to-understand areas
- [ N ] I have made corresponding changes to the documentation
- [ N  ] My changes generate no new errors
- [ N ] I have added tests that prove my fix is effective or that my feature works
- [ N ] New and existing unit tests pass locally with my changes
